go.mod

```diff
@@ -54,6 +54,7 @@ require (
 	github.com/prometheus/common v0.65.0
 	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.34
 	github.com/sirupsen/logrus v1.9.3
+	github.com/spf13/cobra v1.9.1
```

Comment:
🤔 Why there are those changes on `go.mod` ? I see nothing related with changes in go source code. 
Would you please explain or remove this change ?

Reply:
I guess that's the content after I run `go mod tidy` (in `go version go1.25.3 darwin/arm64`)? Should it make sense to keep that output?

Reply:
The dependency was moved from indirect to direct. I guess it does not matter. 
I think the best would be to drop the changes from go.mod and go.sum for this PR